src/helpers/helpers.ts

```diff
@@ -0,0 +1,66 @@
+import { Person } from '../types';
+import { SortType } from '../types/SortType';
+
+export const getPeopleWithParents = (people: Person[]) => {
+  return people.map(person => ({
+    ...person,
+    mother: people.find(woman => woman.name === person.motherName) || null,
+    father: people.find(man => man.name === person.fatherName) || null,
+  }));
+};
+
+export const getFilteredPeople = (
+  people: Person[],
+  sex: string,
+  query: string,
+  centuries: string[],
+  sortBy: string,
+  order: string,
+) => {
```

Comment:
When you have more than 3 arguments for the function it's better to pass them as an object. By this, you don't need to care about the order of the fields, and in your example, it's really hard with 6 fields.
```suggestion
type Props = {
  people: Person[],
  sex: string,
  query: string,
  centuries: string[],
  sortBy: string,
  order: string,
}

export const getFilteredPeople = ({
  people,
  sex,
  query,
  centuries,
  sortBy,
  order,
}) => {
```